web/client/components/charts/__tests__/SimpleChart-test.jsx

```diff
@@ -74,4 +74,113 @@ describe('SimpleChart component', () => {
         expect(tooltip).toExist();
 
     });
+    it('test with y axis vlaues over trillion ', () => {
+        let modifiedData = data;
+        modifiedData[6].uv = 10000000000000;
+
+        ReactDOM.render(<SimpleChart data={modifiedData} type="line" xAxis={{dataKey: "name"}} yAxis series={SERIES}/>, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const el = container.querySelector('div');
+        expect(el).toExist();
+        const maxValue = document.querySelector('#container > div > svg > g.recharts-layer.recharts-y-axis > g > g > g:nth-child(5) > text > tspan').textContent;
+        expect(maxValue).toBe('10 T');
+    });
+    it('test y axis with short strings ', () => {
+        let modifiedData = data;
+        modifiedData.map(y => y.uv = 'string');
```

Comment:
This is not a copy. `modifiedData` and `data` will be the same. This does what you want
```
const modifiedData = data.map(y => {...y, uv: 'string');
```